api/types/role_test.go

```diff
@@ -641,6 +641,49 @@ func TestUnmarshallCreateHostUserModeYAML(t *testing.T) {
 	}
 }
 
+func TestUnmarshallCreateDatabaseUserModeJSON(t *testing.T) {
+	for _, tc := range []struct {
+		expected CreateDatabaseUserMode
+		input    any
+	}{
+		{expected: CreateDatabaseUserMode_DB_USER_MODE_UNSPECIFIED, input: "\"\""},
```

Comment:
Tip: you can use `` for literal strings (no need to escape):

```suggestion
		{expected: CreateDatabaseUserMode_DB_USER_MODE_UNSPECIFIED, input: `""`},
```

Same for others.